lib/client/src/services/ResourceLoaderService.js

```diff
@@ -59,11 +61,31 @@ wilson.service('ResourceLoaderService', ['$q', '$templateCache', function($q, $t
     }
 
 
+  /**
+   *
+   * Loads a the styles for a given id and styleContent. Loading is synchronous because the content is simple
+   * put into the document in a style block.
+   *
+   * @param id
+   * @param styleContent
+   */
+  function loadStyles(id, styleContent) {
+      if (!_styleCache[id]) {
+        if (!_styleElem) {
```

Comment:
Is there no way to do this ahead of time so this method doesn't have to do this type of logic to add if needed for both the styleCache and the element housing the style scripts?